packages/migrations/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "private": true,
+    "name": "@0xproject/migrations",
+    "version": "0.0.1",
+    "description": "0x smart contract migrations",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "scripts": {
+        "build:watch": "tsc -w",
+        "build": "tsc",
+        "clean": "shx rm -rf lib",
+        "lint": "tslint --project . 'src/**/*.ts'",
+        "migrate": "run-s build compile script:migrate",
+        "script:migrate": "node ./lib/migrate.js",
+        "copy_artifacts": "copyfiles 'src/artifacts/**/*' ./lib",
+        "compile": "node ../deployer/lib/src/cli.js compile --contracts ${npm_package_config_contracts} --contracts-dir ../contracts/src/contracts --artifacts-dir src/artifacts"
+    },
+    "config": {
+        "contracts": "Exchange,DummyToken,ZRXToken,Token,WETH9,TokenTransferProxy,MultiSigWallet,MultiSigWalletWithTimeLock,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,MaliciousToken,TokenRegistry,Arbitrage,EtherDelta,AccountLevels"
```

Comment:
Hmm a developer needs to keep this config and the compile command in sync with the contracts repo. It's also in the migration package as an enum. hmm I don't have a solution but I think it's worth pointing out.

Reply:
Agreed that it's a bit brittle at the moment. Will need to be refactored eventually.